src/components/SiteNote.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+const MAX_CHAR = 240;
+
+const ellipsisHelper = note => {
+  return note.length > MAX_CHAR ? note.substr(0, MAX_CHAR) + '...' : note;
+};
+
+const SiteNote = ({ loadedSiteNote }) => {
+  const tempNote = `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur unde
```

Comment:
One thing you should look into is "Default values" when destructuring. It will save you the ternary operator